hydra.yaml

```diff
@@ -0,0 +1,60 @@
+package:
+  name: hydra
+  version: 2.3.0
+  epoch: 0
+  description: OpenID Certified OAuth 2.0 Server and OpenID Connect Provider
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - go
+  environment:
+    CGO_ENABLED: "1" # Needed for SQLite support
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/ory/hydra
+      tag: v${{package.version}}
+      expected-commit: ee8c339ddada3a42529c0416897abc32bad03bbb
+
+  - runs: |
+      go install -tags sqlite,json1,hsm \
```

Comment:
need to change to use pipeline

Reply:
any links to thread where the issue of not using the pipeline is discussed?

Reply:
I think I got this this time